lib/resources/dstu2/soarian_allergy_intolerance.yaml

```diff
@@ -0,0 +1,346 @@
+    {
+      "contained": [
+        {
+          "resourceType": "Practitioner",
+          "id": "123",
+          "practitionerRole": [
+            {
+              "role": {
+                "coding": [
+                  {
+                    "system": "http://hl7.org/fhir/v2/0286",
+                    "code": "RP"
+                  }
+                ]
+              }
+            }
+          ]
+        }
+      ]
+    }
+  note: >
+    These should be either `Practitioner` or `RelatedPerson` resources that are referenced with the [reporter] field.
+  url: http://hl7.org/fhir/DSTU2/domainresource-definitions.html#DomainResource.contained
+
+- name: onset
+  required: 'No'
+  cardinality: 0..1
+  type: dateTime
+  description: Date(/time) when manifestations showed.
+  example: |
+    {
+      "onset: "2012-07-13T00:00:00Z"
+    }
+
+- name: recordedDate
+  required: 'Yes'
+  cardinality: 0..1
+  type: dateTime
+  description: When the sensitivity was recorded.
+  example: |
+    {
+      "recordedDate": "2015-10-14T13:13:20-06:00"
+    }
+
+- name: recorder
+  required: 'No'
+  cardinality: 0..1
+  type: Reference (Practitioner)
+  description: Who recorded the sensitivity.
+  note: Recorder must be a `Practitioner` reference.
+  example: |
+    {
+      "recorder": {
+        "reference": "Practitioner/21500971"
+      }
+    }
+
+- name: patient
+  required: 'Yes'
+  cardinality: 1..1
+  type: Reference (Patient)
+  description: Who the sensitivity is for.
+  example: |
+    {
+      "patient": {
+        "reference": "Patient/5366327"
+      }
+    }
+
+- name: reporter
+  required: 'Yes'
+  cardinality: 0..1
+  type: Reference (Patient) | contained Reference (Practitioner | RelatedPerson)
+  description: Source of the information about the allergy.
+  example: |
+    Reporter as a Patient Reference
+    {
+      "reporter": {
+        "reference": "Patient/5366327"
+      }
+    }
+    
+    Reporter as a contained reference
+    {
+      "reporter": "#5366328"
+    }
+  note: >
+    If [reporter] is a contained reference, a matching `RelatedPerson` or `Practitioner` reference must be provided in
+    the [contained] field.
+
+- name: substance
+  required: 'Yes'
+  cardinality: 1..1
+  type: CodeableConcept
+  description: Substance, (or class) considered to be responsible for risk
+  example: |
+    {
+      "substance": {
+        "coding": [
+          {
+            "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+            "code": "4125",
+            "display": "Ethiodized oil"
+          }
+        ]
+      }
+    }
+  binding:
+    description: Type of the substance and negation codes for reporting no known allergies.
+    terminology:
+    - display: RxNorm
+      system: http://www.nlm.nih.gov/research/umls/rxnorm
+      note: RxNorm is supported for Allergens of medication category.
+    - display: SNOMED CT
+      system: http://snomed.info/
+      note: SNOMED CT is supported for negations.
```

Comment:
Yes. It is only for negations and we clarified by adding text under the header of terminology bindings section